app/assets/javascripts/components/image-cropper.js

```diff
@@ -0,0 +1,78 @@
+//= require cropperjs/dist/cropper.js
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function ImageCropper ($imageCropper) {
+    this.$imageCropper = $imageCropper
+    this.$image = this.$imageCropper.querySelector('.app-c-image-cropper__image')
+  }
```

Comment:
It might be nice to define the 960x640 dimensions as constants/variables so they're a bit more obvious to the casual reader. Right now they're hardcoded as [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)).

```suggestion
  function ImageCropper ($imageCropper) {
    this.$imageCropper = $imageCropper
    this.$image = this.$imageCropper.querySelector('.app-c-image-cropper__image')
    this.$targetWidth = 960
    this.$targetHeight = 640
  }
```